jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalTopicModificationService.java

```diff
@@ -323,8 +321,7 @@ public void updateTopic(Topic topic, Poll poll) {
      * @param currentUser current user
      */
     private void subscribeOnTopicIfNotificationsEnabled(Topic topic, JCUser currentUser) {
```

Comment:
If this method is used only in one single place, let's inline its content into that place.